mill-tpolecat/src/io/github/davidgregory084/Tpolecat.scala

```diff
@@ -1,80 +1,10 @@
-    ScalacOption(Seq("-Ywarn-unused:locals"), isSupported = version => v212 <= version && version < v213),                        // Warn if a local definition is unused.
-    ScalacOption(Seq("-Wunused:locals"), isSupported = version => v213 <= version && version < v300),                             // ^ Replaces the above
-    ScalacOption(Seq("-Ywarn-unused:params"), isSupported = version => v212 <= version && version < v213),                        // Warn if a value parameter is unused.
-    ScalacOption(Seq("-Wunused:params"), isSupported = version => v213 <= version && version < v300),                             // ^ Replaces the above
-    ScalacOption(Seq("-Ywarn-unused:patvars"), isSupported = version => v212 <= version && version < v213),                       // Warn if a variable bound in a pattern is unused.
-    ScalacOption(Seq("-Wunused:patvars"), isSupported = version => v213 <= version && version < v300),                            // ^ Replaces the above
-    ScalacOption(Seq("-Ywarn-unused:privates"), isSupported = version => v212 <= version && version < v213),                      // Warn if a private member is unused.
-    ScalacOption(Seq("-Wunused:privates"), isSupported = version => v213 <= version && version < v300),                           // ^ Replaces the above
-    ScalacOption(Seq("-Ywarn-value-discard"), isSupported = _ < v213),                                                            // Warn when non-Unit expression results are unused.
-    ScalacOption(Seq("-Wvalue-discard"), isSupported = version => v213 <= version && version < v300),                             // ^ Replaces the above
-    ScalacOption(Seq("-Ykind-projector"), isSupported = v300 <= _),                                                               // Enables a subset of kind-projector syntax (see https://github.com/lampepfl/dotty/pull/7775)
-    ScalacOption(Seq("-Vimplicits"), isSupported = version => ScalaVersion(2, 13, 6) <= version && version < v300),               // Enables the tek/splain features to make the compiler print implicit resolution chains when no implicit value can be found
-    ScalacOption(Seq("-Vtype-diffs"), isSupported = version => ScalaVersion(2, 13, 6) <= version && version < v300),              // Enables the tek/splain features to turn type error messages (found: X, required: Y) into colored diffs between the two types
-    ScalacOption(Seq("-Ypartial-unification"), isSupported = version => ScalaVersion(2, 11, 9) <= version && version < v213)      // Enable partial unification in type constructor inference
-  )
 
-  def scalacOptionsFor(scalaVersion: String): Seq[String] = {
-    val scalaVer = ScalaVersion(scalaVersion)
-    allScalacOptions.filter(_.isSupported(scalaVer)).flatMap(_.names)
-  }
+  def scalacOptionsFor(scalaVersion: String): Seq[String] =
+    ScalacOptions.defaultTokensForVersion(ScalaVersion.unsafeFromString(scalaVersion))
```

Comment:
There is no point keeping this file to me. We can just move this in `TpolecatModule.scala`